components/BottomNavigation/tests/unit/BottomNavigationTests.m

```diff
@@ -282,6 +282,28 @@ - (void)testCustomElevation {
   XCTAssertEqualWithAccuracy(self.bottomNavBar.elevation, customElevation, 0.001);
 }
 
+- (void)testDefaultShadowColor {
+  // Then
+  XCTAssertEqualObjects(self.bottomNavBar.shadowColor, UIColor.blackColor);
+  XCTAssertEqual(self.bottomNavBar.layer.shadowColor, UIColor.blackColor.CGColor,
```

Comment:
Nit: CGColor does not have a `description` method for printing as a string so it's probably best to convert it to a `UIColor`.  Converting to `UIColor` means the convenient `XCTAssertEqualObjects()` can be used.   `CGColorEqualToColor()` is also available for color comparison if you want to stick with `CGColor` directly.

I would suggest using UIColor for the assertion:

```objc
XCTAssertEqualObjects([UIColor colorWithCGColor:self.bottomNavBar.layer.shadowColor], UIColor.blackColor);
```

Reply:
Updated to use CGColorEqualToColor.